awx/ui_next/src/screens/Application/ApplicationsList/ApplicationListItem.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { string, bool, func } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import {
+  Button,
+  DataListAction as _DataListAction,
+  DataListCheck,
+  DataListItem,
+  DataListItemCells,
+  DataListItemRow,
+  Tooltip,
+} from '@patternfly/react-core';
+
+import { t } from '@lingui/macro';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { PencilAltIcon } from '@patternfly/react-icons';
+import { Application } from '../../../types';
+import DataListCell from '../../../components/DataListCell';
+
+const DataListAction = styled(_DataListAction)`
+  align-items: center;
+  display: grid;
+  grid-gap: 16px;
+  grid-template-columns: 40px;
+`;
+
+function ApplicationListItem({
+  application,
+  isSelected,
+  onSelect,
+  detailUrl,
+  i18n,
+}) {
+  ApplicationListItem.propTypes = {
```

Comment:
Normally we put this outside (underneath) the function - did we change that pattern?

Reply:
I do think I've seen it in a couple of different locations, but I agree, this should be beneath the function. At least that is where I prefer it.